CMakeLists.txt

```diff
@@ -152,6 +152,30 @@ catkin_package(
 #   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 # )
 
+# Mark executable scripts (Python etc.) for installation
+# in contrast to setup.py, you can choose the destination.
+#
+# We search for script files using the FILE(GLOB ...) macro, then assign the
+# result to openag_script_files.
+#
+# http://docs.ros.org/kinetic/api/catkin/html/howto/format2/installing_python.html
```

Comment:
Excellent.  Thank you @gordonbrander for commenting with the sources you used to figure this out.